asm/vm/asm_vm

```diff
@@ -518,62 +536,203 @@ EOF
   fi
 }
 
-validate_asm_installation() {
+validate_retrieve_asm_installation() {
```

Comment:
What does this function name mean?

Reply:
It also retrieves the asm revisions. But renamed.